core/src/test/java/com/netflix/iceberg/hadoop/TestHadoopCommits.java

```diff
@@ -242,4 +246,50 @@ public void testMergeAppend() throws Exception {
     Assert.assertEquals("Current snapshot should contain 1 merged manifest",
         1, metadata.currentSnapshot().manifests().size());
   }
+
+  @Test
+  public void testSchemaUpdateComplexType() throws Exception {
```

Comment:
Minor: How about you add this method right below existing `testSchemaUpdate()` test case? To me, they’re closely related.

Reply:
+1. Relocated test case below existing `testSchemaUpdate()`